client/state/data-layer/wpcom/sites/rewind/index.js

```diff
@@ -27,17 +33,75 @@ const fetchRewindState = action =>
 		action
 	);
 
+/**
+ * Re-requests the activity log entries for all
+ * previously-sent queries in application state
+ *
+ * This function is highly coupled to the details
+ * of Activity Log and Rewind state!
+ *
+ * It will not grab events we haven't already
+ * gotten. Instead it will reissue the same reqeusts
```

Comment:
reqeusts --> requests

Reply:
I wish I could claim that these are sentinels to see if anyone reads the comments in review but that's not true.

Reply:
fixed in b2e521c455